components/brave_rewards/common/mojom/bat_ledger.mojom

```diff
@@ -1,52 +1,68 @@
 // Copyright (c) 2019 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
-// You can obtain one at http://mozilla.org/MPL/2.0/.
-module bat_ledger.mojom;
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+module ledger.mojom;
```

Comment:
I'd do this in the follow-up mentioned above. Since `brave_rewards` is already in the path, would `rewards.mojom` work for you?

Reply:
Sorry, I was thinking about the generated C++ namespace instead of the file name. If we are going to move everything into the `brave_rewards` namespace, could we just put this into the `brave_rewards::mojom` namespace now?

Reply:
The problem with doing that in this PR is that we have this pattern:
```
namespace ledger {
  ... `mojom::something`
}
```
Above works because of `module ledger.mojom;` (since `mojom` is a nested namespace of `ledger`). If I changed this now to `module brave_rewards.mojom;`, I'd have to replace the above usages with
```
namespace ledger {
  ... `brave_rewards::mojom::something`
}
```
unless I changed the `ledger` namespace everywhere to `brave_rewards`, which would explode this PR even more.

Reply:
Also, ideally, namespace names should be short. `brave_rewards` is very long. Any particular reason why we're not considering `rewards`?

Reply:
Your point above makes sense 👍